.storybook/config.js

```diff
@@ -1,10 +1,11 @@
-import { withBackgrounds } from "@storybook/addon-backgrounds";
+import { withContexts } from '@storybook/addon-contexts/react';
 import { withInfo } from "@storybook/addon-info";
 import { withKnobs } from "@storybook/addon-knobs";
 import { addDecorator, addParameters, configure } from "@storybook/react";
 import { StenaTheme } from "./stena-theme";
+import { contexts } from './configs/contexts';
 
-import { withA11y } from "@storybook/addon-a11y"; // Must be imported last, otherwise we get weird error.
+import { withA11y } from "@storybook/addon-a11y";
```

Comment:
Is this no longer an issue?
This error was really hard to find, so if it is still an issue, I think the comment should be kept.

Reply:
Don't know. I revert!